.github/workflows/benchmark_nightly_gpu.yml

```
@@ -0,0 +1,25 @@
+name: Benchmark torchserve gpu nightly
+
+on:
+  # run every day at 2:15am
+  schedule:
+    - cron:  '15 02 * * *'
+jobs:
+  nightly:
+    runs-on: self-hosted
+    steps:
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+          architecture: x64
+      - name: Checkout TorchServe
+        uses: actions/checkout@v2
+      - name: Install dependencies
+          run: |
+            apt-get update
+            apt-get install sudo -y
+            sudo apt-get install apache2-utils
+            pip install -r benchmarks/requirements-ab.txt
```

This action has helped me a lot when installing java
```suggestion
            pip install -r benchmarks/requirements-ab.txt
     - name: Setup Java 11
        uses: actions/setup-java@v2
        with:
          distribution: 'zulu'
          java-version: '11'
```